misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/variable/VariableFactory.java

```diff
@@ -166,6 +167,8 @@ public static BVariable getVariable(SuspendedContext context, String varName, Va
             return new BHandle(context, varName, value);
         } else if (valueTypeName.equals(JVMValueType.STREAM_VALUE.getString())) {
             return new BStream(context, varName, value);
+        } else if (valueTypeName.equals(JVMValueType.REGEXP_VALUE.getString())) {
```

Comment:
Since `JVMValueType` is an enum can't we do like this:
```java
} else if (JVMValueType.REGEXP_VALUE.equals(valueTypeName)) {
```

Reply:
Yes we can do that too. But some of these if statements are using `contains()` checks, so if we use the suggested pattern that might get inconsistent with the common pattern used within the variable factory. Therefore we can keep the existing pattern ftm, IMO.

Reply:
Got your point. I still think we should have structure this in a different way. But, it is out of the scope of this PR. 
+1